storage/connectors/redis/src/main/java/feast/storage/connectors/redis/writer/RedisFeatureSink.java

```diff
@@ -60,29 +71,42 @@ public static Builder builder() {
   public abstract static class Builder {
     public abstract Builder setRedisConfig(RedisConfig redisConfig);
 
+    public abstract Builder setRedisClusterConfig(RedisClusterConfig redisConfig);
+
     public abstract Builder setFeatureSetSpecs(Map<String, FeatureSetSpec> featureSetSpecs);
 
     public abstract RedisFeatureSink build();
   }
 
   @Override
   public void prepareWrite(FeatureSet featureSet) {
-
-    RedisClient redisClient =
-        RedisClient.create(RedisURI.create(getRedisConfig().getHost(), getRedisConfig().getPort()));
-    try {
-      redisClient.connect();
-    } catch (RedisConnectionException e) {
-      throw new RuntimeException(
-          String.format(
-              "Failed to connect to Redis at host: '%s' port: '%d'. Please check that your Redis is running and accessible from Feast.",
-              getRedisConfig().getHost(), getRedisConfig().getPort()));
+    if (getRedisConfig() != null) {
```

Comment:
Should we handle the null case?

Reply:
Not sure, how exactly. I just mimiced the behavior of RedisFeatureSink and RedisClusterFeatureSink. So the latest do nothing in this method. So when RedisConfig is null I do the same.
Do you want to have exception here?

Reply:
added throw